archive/r/ruby/fibonacci.rb

```diff
@@ -0,0 +1,21 @@
+def fibonacci(number)
+    a = 0
+    b = 1
+    c = 0
+
+    i = 0
+    while i < number
+        c = a + b
+        b = a
+        a = c
+
+        print("#{c}\n")
```

Comment:
Can you have this also print the index? I think that's the current standard at the moment:

```
1: 1
2: 1
3: 2
4: 3
5: 5
```